src/transform/AstDumpDotPass.cpp

```diff
@@ -207,6 +216,49 @@ void AstDumpDotVisitor::visit( Specification& specification )
     m_stream << "}\n";
 }
 
+void AstDumpDotVisitor::visit( HeaderDefinition& node )
+{
+    DotLink link( this, &node );
+    dumpNode( node, "HeaderDefinition" );
+    RecursiveVisitor::visit( node );
+}
+
+void AstDumpDotVisitor::visit( InitDefinition& node )
+{
+    DotLink link( this, &node );
+    dumpNode( node, "InitDefinition" );
+    node.initToken()->accept( *this );
```

Comment:
Use `RecursiveVisitor::visit( node )` instead of the accepts. The dot dump should reflect the AST + recursive visitation as is.